docker/Dockerfile

```diff
@@ -14,10 +14,15 @@ RUN apt-get update; \
 
 WORKDIR /opt
 
-COPY package.json package-lock.json* ./
+COPY package.json package-lock.json* tsconfig.json copy-assets.js ./
+COPY src ./src
 
-RUN npm ci --${NODE_ENV}
-RUN npm cache clean --force
+RUN if [ "$BUILD_ENV" = "production" ]; \
+    then npm ci && npm prune --production; \
```

Comment:
What we used before: `npm ci --${NODE_ENV}` has actually been wrong, and the env was ignored.

Reply:
I believe `prod` and `non prod` build should be the same except that in prod build we push image to the remote repo. This shenanigans make build complex and me sad